src/components/Home.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { CiSearch } from 'react-icons/ci';
+import { BsArrowLeftCircle, BsMicFill } from 'react-icons/bs';
+import { IoMdSettings } from 'react-icons/io';
+import { NavLink } from 'react-router-dom';
+import { getCities, searchCity } from '../redux/cities/citiesSlice';
+import City from './City';
+
+function Home() {
+  const dispatch = useDispatch();
+  const cities = useSelector((state) => state.cities.cities);
+  const loading = useSelector((state) => state.cities.loading);
+  const error = useSelector((state) => state.cities.error);
+  const searchedCity = useSelector((state) => state.cities.searchedCity);
+
+  useEffect(() => {
+    if (cities.length === 0) {
+      dispatch(getCities());
+    }
+  }, [dispatch, cities.length]);
+
+  if (loading) {
+    return <h3>Loading...</h3>;
+  }
+
+  if (error) {
+    return <h3>Something went wrong ...</h3>;
+  }
+
+  // eslint-disable-next-line max-len
+  const filteredCities = searchedCity
+    ? cities.filter((city) => {
+      const cityName = city.city_name.toLowerCase();
+      const searchValue = searchedCity.toLowerCase();
+      return cityName.includes(searchValue);
+    })
+    : cities;
+
+  return (
+    <div>
```

Comment:
[OPTIONAL] while you did an impressive job working on the project requirements I NOTICED YOU disabled eslint in your Home.js file it is against microverse policy to disable eslint.  it would be nice if you enabled eslint